### ISSUE TITLE
Successive Episode

### ISSUE THREAD

**Issue by Alfa-beto** [Stream4me/addon#181]
Hi, a greeting to the KOD team. 

I'm part of the Alfa development team and would like to try to adapt your successive episode code to our addon if you don't mind. 
I have a few questions since I've seen some of the code. Does this feature accept multi-channel? i.e. if a series has strms from two or more channels this code has no problem with it?

**Comment by Alhaziel01** [comment]
Hello Alfa-beto,

we are pleased that this feature interests you.
From the tests we did it has no problem with the multichannel.
Next episode only sends the next .strm file to play_from_library.

**Comment by Alfa-beto** [comment]
Thanks for the answer @Alhaziel01 , it is an interesting and useful function, we will try to adapt the code. 
We would also like to thank the kod team for their collaboration with Alfa. 
Greetings and if you need anything you know where to contact us.

**Comment by Alhaziel01** [comment]
We are always happy to collaborate but, sometimes, due to lack of time, differences with Alfa or more simply because we don't know if you need it (such as, for example, some servers or DNS Resolver), we can't always make a pull request of the implementations that we add to KoD.
Feel free to use the code you need, it's open source.

**Comment by Alhaziel01** [comment]
We close this tread since nextep porting has been completed.
The idea of ​​the thumbnail is good, we decided to implement it in KoD.

P.S. thanks for adding py3 support we started the code rebase.

Last question, where do you prefer to be contacted, on github or on the forum?

**Comment by Alfa-beto** [comment]
Hello. 

Yes, we have integrated the code of nextep, after giving it a little bit of love, we are glad you liked the modifications we made and we hope your users appreciate it as much as ours.

About the collaboration, your PRs will always be welcome on github. Especially in the case of the servers that we already know tend to change often.

Regarding the contact it is better to contact privately in the forum.

The conversion to python 3 is being done little by little any comment or suggestion will be welcome too.

Again thanks for sharing the next episode code with us.
